invisible_cities/cities/detsim_simulate_electrons_test.py

```diff
@@ -0,0 +1,78 @@
+import os
+import numpy  as np
+import tables as tb
+import pandas as pd
+
+from pytest import fixture
+
+from invisible_cities.core.configure import configure
+
+from invisible_cities.io.mcinfo_io import read_mchits_df
+
+from invisible_cities.cities.detsim_simulate_electrons import *
+
+
+@fixture(params = ["Kr83_nexus_v5_03_00_ACTIVE_7bar_3evts.MCRD.h5"])
+def nexusMC_filename(request, ICDATADIR):
+    return os.path.join(ICDATADIR, request.param)
+
+
+@fixture
+def random_event_hits(nexusMC_filename):
+    filename = nexusMC_filename
+
+    with tb.open_file(filename) as h5in:
+        extents = pd.read_hdf(filename, 'MC/extents')
+        event_ids  = extents.evt_number
+        hits_df    = read_mchits_df(h5in, extents)
+
+        #choose random event
+        evt = np.random.choice(event_ids)
+        hits = hits_df.loc[evt, :, :]
+
+        print("Event", evt)
+
+        xs = hits["x"].values
+        ys = hits["y"].values
+        zs = hits["z"].values
+        energies = hits["energy"].values
+
+    return xs, ys, zs, energies
+
+
+@fixture
+def parameters():
+
+    conf = configure('detsim invisible_cities/config/detsim.conf'.split())
+
+    print(conf)
+
+    wi, fano_factor          = conf.get("wi")      , conf.get("fano_factor")
+    lifetime, drift_velocity = conf.get("lifetime"), conf.get("drift_velocity")
+    transversal_diffusion    = conf.get("transversal_diffusion")
+    longitudinal_diffusion   = conf.get("longitudinal_diffusion")
+
+    return wi, fano_factor, lifetime, drift_velocity,\
+           transversal_diffusion, longitudinal_diffusion
```

Comment:
There are a lot of fixtures here for something that could probably be one and just have the values written into them rather than read from a config.

I'd think a bit more about the scope of the information you need for your test (and maybe others) and write it into a fixture of similar format to those already in IC. If you think it's something that could be useful in its entirety or partially in other tests in other files, we can put it in a conftest.py file and set the scope accordingly, if you think it's just for this module put it here and set the scope to the module.